symphony/bdk/core/client/__init__.py

```diff
@@ -1,2 +1,5 @@
 """Package for instantiating internal API clients
 """
+from symphony.bdk.core.client.trace_id import setup_trace_id_log_record_factory
+
+setup_trace_id_log_record_factory()
```

Comment:
just for my knowledge when will this be called? whenever a member of this package is loaded?

Reply:
Yes, __init__ is run first time we import this package or any child package.